antd-slinky/src/main/scala/demo/AntdFacade.scala

```diff
@@ -0,0 +1,88 @@
+  val Spin:      ExternalComponentP[SpinProps]      = importSTComponent(Antd.Spin)
+  val Tag:       ExternalComponentP[TagProps]       = importSTComponent(Antd.Tag)
+
+  /* export unchanged */
+  @inline def AlertProps = typings.antdLib.esAlertMod.AlertProps
+  type AlertProps = typings.antdLib.esAlertMod.AlertProps
+  @inline def ButtonProps = typings.antdLib.esButtonButtonMod.ButtonProps
+  type ButtonProps = typings.antdLib.esButtonButtonMod.ButtonProps
+  @inline def ColProps = typings.antdLib.esGridColMod.ColProps
+  type ColProps = typings.antdLib.esGridColMod.ColProps
+  @inline def ColumnProps = typings.antdLib.esTableInterfaceMod.ColumnProps
+  type ColumnProps[T] = typings.antdLib.esTableInterfaceMod.ColumnProps[T]
+  @inline def FormItemProps = typings.antdLib.esFormFormItemMod.FormItemProps
+  type FormItemProps = typings.antdLib.esFormFormItemMod.FormItemProps
+  @inline def FormProps = typings.antdLib.esFormFormMod.FormProps
+  type FormProps = typings.antdLib.esFormFormMod.FormProps
+  @inline def IconProps = typings.antdLib.esIconMod.IconProps
+  type IconProps = typings.antdLib.esIconMod.IconProps
+  @inline def InputProps = typings.antdLib.esInputInputMod.InputProps
+  type InputProps = typings.antdLib.esInputInputMod.InputProps
+  @inline def ModalProps = typings.antdLib.esModalModalMod.ModalProps
+  type ModalProps = typings.antdLib.esModalModalMod.ModalProps
+  @inline def OptionProps = typings.antdLib.esSelectMod.OptionProps
+  type OptionProps = typings.antdLib.esSelectMod.OptionProps
+  @inline def PasswordProps = typings.antdLib.esInputPasswordMod.PasswordProps
+  type PasswordProps = typings.antdLib.esInputPasswordMod.PasswordProps
+  @inline def RowProps = typings.antdLib.esGridRowMod.RowProps
+  type RowProps = typings.antdLib.esGridRowMod.RowProps
+  @inline def SelectProps = typings.antdLib.esSelectMod.SelectProps
+  type SelectProps[T] = typings.antdLib.esSelectMod.SelectProps[T]
+  @inline def SpinProps = typings.antdLib.esSpinMod.SpinProps
+  type SpinProps = typings.antdLib.esSpinMod.SpinProps
+  @inline def TableProps = typings.antdLib.esTableInterfaceMod.TableProps
+  type TableProps[T] = typings.antdLib.esTableInterfaceMod.TableProps[T]
+  @inline def TagProps = typings.antdLib.esTagMod.TagProps
+  type TagProps = typings.antdLib.esTagMod.TagProps
+
+  /**
+    * This is an example of something a bit more complicated than just rewriting component types, and which a manually
+    *  written facade. Given an implementation of a component which has a `form` prop which is to be prefilled,
+    *  this will generate a ready-to-use `ExternalComponent` for it.
+    */
+  def formComponent[Props <: js.Object](
+      options: FormCreateOption[Props]
+  )(f:         js.Function1[Props with WithForm, ReactElement]): ExternalComponentP[Props] =
+    importSTComponent(createForm(FormCreateOption[Props](name = "coordinated"))(f).asInstanceOf[ComponentType[Props]])
```

Comment:
I wonder if `name` shouldn't be a param of  `def formComponent`, to avoid conflicts between multiple forms (in the same page?)

Reply:
Not sure. It has a `FormCreateOption` has a few other things, i have no idea if they are useful

Reply:
Ok, you are right, but then in this case in the function the `options` param must be utilized, I changed this in https://github.com/oyvindberg/ScalablyTypedDemos/pull/10/commits/8a116ed8bb3b1e0b144989524a4c0e998d980fa9